torch/ao/quantization/fx/_model_report/detector.py

```diff
@@ -938,6 +938,9 @@ class OutlierDetector(DetectorBase):
         reference_percentile (float, optional): The denominator to find the relative scale of the 100th percentile
             Should be between 0 and 1
             Default: 0.975
+        statistical_threshold (float, optional): Fraction of batches to determine outliers per channel should be above this
+            Should be between 0 and 1
+            Default: 0.95
```

Comment:
Hmm, should this be configurable? What is a potential use case?

Reply:
Also this comment currently sounds like an exception will be thrown if the fraction is below this, but I don't think that's the case. Maybe add what happens if the fraction is below this, e.g. the result will be marked as "insignificant" or something

Reply:
I see. To me that still sounds like a bit of a niche use case and I'm not sure if people would actually use it, but OK to keep this as is. One risk in making too many things configurable is an overly complicated API and extra backward compatibility burden.